docs/reference/connectors/gpubsub_producer.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_label: gpubsub_producer (Google Pub/Sub)
+sidebar_position: 1
+---
+
+# The `gpubsub_consumer` Connector
+
+This connector allows producing to a Google PubSub queue.
+
+## Configuration
+
+The credentials must be provided in a JSON file. The path to the JSON file should be set as an environment variable called `GOOGLE_APPLICATION_CREDENTIALS`
```

Comment:
This should be updated to note that it also reads from wherever gcloud expects stuff and works magically when deployed on google machines